src/main/resources/messages.properties

```diff
@@ -16,10 +16,10 @@ invalid.username-or-password=Please enter valid email/login or password
 invalid.phone.number=Phone number has to contain your country code and no letters (Ex. +380501234567 or +1(234)567-89-10)
 invalid.phones-amount=A user can't have more than {max} registered phones
 invalid.first-or-last.name=${validatedValue} must be between 2 and 50 characters (allowed only ['] and [-] symbols)
-invalid.confirm.email=Invalid confirmation of email. Make sure the new email matches the confirmation of email
+invalid.confirm.email=Invalid confirmation of email. The new email matches the confirmation of email
```

Comment:
You changed the meaning, please make it correct: "The new email has to match the conformation email"